## Performing Binomial Distribution
 X-Ray taken in the Radiology department, 200 X-rays images taken per/day. Due to the instrumentation failure probability of defective image X-Ray or Noise Image X-Ray is 2% what is the probability of selecting the two or more Noise image X-ray. Find the suitable probability

In [ ]:
#total x-ray taken per day
n = 200

#probability of defective/ noisy images
p = 0.02   #(Consider it probability of success)

We need to find the probability of selecting the two or more Noise image X-ray. Which can be denoted as P(X>=2)

In [ ]:
mean = n*p
var = n*p*(1-p)
print("Mean and Variance of the distribution are " + str(mean) + " and " + str(var) + " respectively")

Mean and Variance of the distribution are 4.0 and 3.92 respectively


To find the probability of getting at least two defective images, we can sum the probabilities for k=2 to k=200, or more conveniently, use the complement rule:

P(X>=2) = 1 - P(X<2)

In [ ]:
# Denote p0 as probability of 0 defective image
p0 = (1 - p) ** n
p0

0.0175879466057215

In [ ]:
# Denote p1 as probability of 1 defective image
p1 = n * p * (1 - p) ** (n - 1)
p1

0.07178753716621021

In [ ]:
prob_greater_than_equal_to_2 = 1 - (p0 + p1)
prob_greater_than_equal_to_2

0.9106245162280683

The probability of selecting two or more noise image X-rays out of 200 is approximately 0.9106 (or 91.06%).